### ISSUE TITLE
Per language stats contributions board

### ISSUE THREAD

**Issue by phlax** [translate/pootle#4210]

**Comment by phlax** [comment]
- Per language stats contributions board
- per user rankings per tp

**Comment by iafan** [comment]
Are there any requirements / design considerations for these features yet?

**Comment by dwaynebailey** [comment]
@iafan no, none.  I think our first approach is how to expose the leaderboard that is site wide to languages, projects and TPs.  As well as how to expose the contribution graphs that are currently individual to language, project and TP.

If you have any ideas, they'd be gladly accepted.

**Comment by julen** [comment]
This looks like a dupe of #3932.

**Comment by phlax** [comment]
@julen partly it is but there are other requirements too - ie per language stats

this bug has come about partly from feedback from users while attending Mozfest

before attempting to address it i wanted to get as many suggestions as possible on how best to represent stats, and to work out the priorities from there

**Comment by iafan** [comment]
@dwaynebailey, what are the contribution graphs?

**Comment by dwaynebailey** [comment]
@iafan those would be http://mozilla.locamotion.org/user/dwayne/stats/

**Comment by iafan** [comment]
I think to implement this nicely we will need to update our sidebar logic to allow multiple sidebar tabs / modes. I threw together a prototype that shows two sidebar tabs: one for generic information (stats/graphs/announcements), and one for tools (import/export/etc.). Eventually we can have more tabs (e.g. for chats), we can probably move stats into a sidebar as well (this would simplify the header).

See the demo: http://jsfiddle.net/iafan/62ybc41w/6/

This aligns with the prototypes of the new sidebar I showed earlier as a part of this document: https://docs.google.com/document/d/1aSYNRkgZe_dwuRV43H9D4azq4bpYR6x5bV5T99yYq-E/edit

![sidebar](https://cloud.githubusercontent.com/assets/1728158/11352706/becf54a2-91f2-11e5-98dd-987db77e388d.png)
